programming/sr/vision/index.md

```diff
@@ -171,62 +184,71 @@ One (possibly both) of them may change to resolve this.
 
 An `Orientation` object describes the orientation of a marker.
 
-![A visual representation of how the orientation axes work. Source: SourceBots]({{ site.baseurl }}/images/content/vision/yaw-pitch-roll.png)
-
-pitch
+rot_x
 :   Rotation of the marker about the cartesian x-axis, in radians.
 
-    Leaning a marker away from the camera increases the value of `rot_x`, while
-    leaning it towards the camera decreases it. A value of 0 indicates that the
-    marker is upright.
+    Leaning a marker towards the camera increases the value of `rot_x`, while
+    leaning it away from the camera decreases it. A value of π or -π indicates
+    that the marker is upright.
 
-yaw
+rot_y
 :   Rotation of the marker about the cartesian y-axis, in radians.
 
-    Turning a marker clockwise (as viewed from above) increases the value of
-    `rot_y`, while turning it anticlockwise decreases it. A value of 0 means
+    Turning a marker clockwise (as viewed from above) decreases the value of
+    `rot_y`, while turning it anticlockwise increases it. A value of 0 means
     that the marker is perpendicular to the line of sight of the camera.
 
-roll
+rot_z
 :   Rotation of the marker about the cartesian z-axis, in radians.
 
     Turning a marker anticlockwise (as viewed from the camera) increases the
-    value of `rot_z`, while turning it clockwise decreases it. A value of 0
-    indicates that the marker is upright.
+    value of `rot_z`, while turning it clockwise decreases it. A value of π
+    or -π indicates that the marker is upright.
 
-rot_x
-:   An alias for `pitch`.
+roll
+:   An alias for `rot_x`.
 
-rot_y
-:   An alias for `yaw`.
+pitch
+:   An alias for `rot_y`.
 
-rot_z
-:   An alias for `roll`.
+yaw
+:   An alias for `rot_z`.
```

Comment:
```suggestion
:   A rotation about the about the vertical axis, in radians (An axis top to bottom through the token).
    Turning a marker clockwise (as viewed from above) increases the value of
    `yaw`, while turning it anticlockwise decreases it. A value of 0 means
    that the marker is perpendicular to the line of sight of the camera.
    
    This differs from `rot_y` in the direction that increases the value.
```

Reply:
Is it intentional that this defines the zero point as "perpendicular to the line of sight of the camera" where others frame it as "facing the camera square-on"? Specifically: is there a semantic difference that this is trying to convey?
If not, I think we should use the same framing everywhere.

Reply:
Face-on to the camera implies that the yaw is calculated relative to the tagent of a sphere aroud the camera while it is actually calculated relative to a flat plane in front of the camera.
![](https://user-images.githubusercontent.com/25060060/216167292-55ab9758-3387-4bc0-8816-6d17d376e8fc.png)

Reply:
Ok, can we use the "perpendicular to the line of sight of the camera" for the pitch case too then? Presumably that suffers the same issue, just in a different plane.

Reply:
Yes, this is true for all bits talking about orientation.

Reply:
6de4868